transactions/types.go

```diff
@@ -55,9 +64,17 @@ type SendTxArgs struct {
 	Input types.HexBytes `json:"input"`
 	Data  types.HexBytes `json:"data"`
 
-	// additional data
+	// additional data - version SendTxArgsVersion0
 	MultiTransactionID wallet_common.MultiTransactionIDType
 	Symbol             string
+	// additional data - version SendTxArgsVersion1
+	ValueOut           *hexutil.Big
+	FromChainID        uint64
+	ToChainID          uint64
+	FromTokenID        string
+	ToTokenID          string
+	ToContractAddress  types.Address // represents address of the contract that needs to be used in order to send assets, like ERC721 or ERC1155 tx
```

Comment:
Hmm isn't this what the `To` field contains whenever we're not doing a simple ETH transfer?

Reply:
@dlipicar well, this can be a topic for discussion as well. :)

My point is that we should be as explicit as possible, not think about what is `To` for example a recipient or a contract address, and that's why I added that new field and also plan to update it in one of the later steps the `Transactor` implementation (once mobile and desktop fully moved to that). 

When a client sends a tx from address A to address B, then A and B should be the sender and the recipient respectively, and any other address (contract) that should be used as an intermediary step to achieve that transfer should be set as a contract address field. In that case, the dev would be able to pack/send transactions easily without following the entire flow to figure out is "To" final destination, recipient or contract address.

What we have now, whenever I see the Transaction part I am always confused is the `To` a recipient address or a contract address and never know if packed the data correctly or not without following entire flow.

Reply:
ahh ok, I think I got confused by the reuse of the old fields.

So, in `SendTxArgsVersion0` `To` has the old meaning, in `SendTxArgsVersion1` `To` is the destination address for a transfer. Is that correct? Same for `Value` I guess (only used for ETH transfers in `SendTxArgsVersion0`, used for any transaction in `SendTxArgsVersion1`).
Already mentioned it before, I know it's planned for later, but I'd really like to keep "raw blockchain" fields and "input/interpreted" fields separate...